resources/aur.rb

```diff
@@ -17,13 +17,15 @@
 # limitations under the License.
 #
 
-actions :build, :install
+actions :build, :install, :sync
 
 default_action :install
 
 attribute :package_name, :name_attribute => true
 attribute :version, :default => nil
-attribute :builddir, :default => node[:pacman][:build_dir]
```

Comment:
This is a breaking change, but speaking of those should the `:install` action also install packages with dependencies? That change to `:install` will also be a breaking change, so maybe keep this change too and upgrade to `3.0.0`?
I'm also fine with keeping the old name.